src/Repository/UserRepository.php

```diff
@@ -71,4 +71,16 @@ public function compareTokensWhenChangingPassword(?string $tokenReset): ?User
     {
         return $this->findOneBy(['tokenReset' => $tokenReset]);
     }
+
+    public function showUsersPaginated(array $paginate): array
```

Comment:
please pass 2 variables instead of an array. passing the array makes it look like you are trying to use a data structure of type struct (as in C) and it's not very common/indicated to do this in PHP